### ISSUE TITLE
Can EFA Provide a Global Rule based Filter

### ISSUE THREAD

**Issue by yoke88** [E-F-A/v4#129]
can EFA provide a Rule based Filter to control the whole mail flow , we can create differen policy for diffrent domains or email addresses. the proxmox mail gateway and cisco ironport provide such feature.

![0613967e-4bca-4013-8b4d-6c32e1dcf7f1 png](https://user-images.githubusercontent.com/3146226/52766043-ddf00880-3060-11e9-864d-b04c73754b62.jpg)

**Comment by shawniverson** [comment]
@yoke88 Yes, indeed, it could.  What eFa lacks right now is a GUI to manage rulesets in this fashion.  Out of curiosity, do you think the proxmox mail gateway does a good job the way it makes its rules?

**Comment by yoke88** [comment]
the best mail gateway i have used which does good in rules manage is cisco ironport.

but cisco ironport is commercial software. in the open source or free mail gateways i have used , only proxmox mail gateway provide this feature to let user to manage rule set using GUI, and the gui provides a glue of how the rule is processed . the ruleset management is easy to use and it look like outlook rules or iptables rules , so users like me (not a coder) can get started soon.
